Implement blf-balancing-position-control application and YarpTextLogging python bindings

GiulioRomualdi opened the pull request:
This PR adds:
- the python bindings for the `YarpTextLogging` application
- a utility to move the CoM of the robot on a given trajectory. 

The application will be automatically installed along with the configuration files

S-Dafarra commented on utilities/balancing-position-control/README.md:
```diff
@@ -0,0 +1,26 @@
+following a given trajectory.
+
+## 🏃 How to use the application
+The fastest way to use the utility is to run the `python` application
+[`blf-balancing-position-control.py`](./script/blf-balancing-position-control.py).
+If you correctly installed the framework, you can run the application from any folder.
```
I would suggest adding some more detail on how to run the app, and on the assumptions you have about the robot status. For example, does the robot need to be on the ground? Is the com supposed to be close the first know point? Are the feet kept fixed?

GiulioRomualdi commented:
Hi @S-Dafarra I added the units in the config files and I added `contact_force_threshold` in the configuration file. I also tried to improve the readme.

GiulioRomualdi commented on utilities/balancing-position-control/README.md:
```diff
@@ -0,0 +1,41 @@
+1. **balancing-position-control** does not consider the measured zero moment point (ZMP) to generate
+   the CoM trajectory. This means that a user can also run the application when the robot is placed
+   on external support.
+2. The `com_knots_delta_<>` lists represent the coordinate in the CoM frame at `t=0s`this means
+   that the one may also run the application when the robot is in single support. However, in that
+   case, the user must be sure that the CoM trajectory is always within the support polygon and that
+   the joint tracking performance is sufficient accurate to prevent the robot from falling.
```
I asked to keep the feet in a given configuration. [Here](https://github.com/ami-iit/bipedal-locomotion-framework/blob/1a7926d8967800d1ef4b36e52ad1877fb8d3aa2a/utilities/balancing-position-control/config/robots/ergoCubGazeboV1/balancing_control/ik.ini) the structure of the IK

S-Dafarra replied:
Probably it is worth adding a line in the README as it may not be immediately clear to novice users

S-Dafarra replied:
I would also suggest specifying how the references for the feet and the joint regularization are computed. Are the initial measured values?

GiulioRomualdi replied:
> I would also suggest specifying how the references for the feet and the joint regularization are computed. Are the initial measured values?

Yes I add a line in the readme

GiulioRomualdi replied:
Done https://github.com/ami-iit/bipedal-locomotion-framework/pull/611/commits/f5a1be665856467f9ec14a436649a9972627615d
